docs/UserWallet.rst

```diff
@@ -58,17 +53,30 @@ A warning message appears behind the label "My wallet"
 
 |image5|
 
+Once your wallet is approved by an administrator, you recieve a notification informing you that your wallet is activated.
+
+|image|
+
+All actions become available and you are allowed to send and recieve funds as well as consulting the transactions hoistory, rewarding details and see the transactions statistics.
+
+It is possible to filter ...
```

Comment:
@samehaidi incomplete?

Reply:
I wanted to add a paragraph that is related to statistics capabilities for filters. 
I think it is not the right location to put it.
WDYT?

Reply:
I don’t knowif it is the right location but it can’t stay like this.

Reply:
i deleted it.